src/scripts/sssom_to_robot_template.py

```diff
@@ -0,0 +1,54 @@
+"""Convert SSSOM to ROBOT template."""
```

Comment:
Script for converting SSSOM to ROBOT template

addresses:
- https://github.com/monarch-initiative/mondolib/issues/11